wordpress-mobile/WordPress-Android
# Merge 13.6 release notes into `master`

**To Test:**
- Verify that all tests pass in CI
- Verify that the release notes keys are correct in the `.po` file
- Verify that the release notes make sense and have no typos
